crates/client/src/lib.rs

```diff
@@ -559,6 +554,9 @@ pub enum ClientError {
     #[error("no default registry server URL is configured")]
     NoDefaultUrl,
 
+    /// The operator failed validation.
+    #[error("operator failed validation")]
+    OperatorValidationFailed,
```

Comment:
This should probably store the validation error so it can be printed with the message, e.g. `operator validation failed: {0}`.